src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -0,0 +1,40 @@
+package org.codeforamerica.messaging.models;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.*;
+import org.codeforamerica.messaging.utils.RegexPatternStrings;
+import org.codeforamerica.messaging.validators.ValidMessage;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import lombok.Builder;
+import java.time.OffsetDateTime;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+@ValidMessage
+public class Message {
```

Comment:
Curious why you renamed this from MessageRequest

Reply:
It seemed to me that `MessageRequest` was the same as `Message` but earlier in state, ie before being fulfilled. When we renamed `Message` to `SmsMessage` the name opened up.

Reply:
I'm a little concerned that it introduces confusion since it sounds singular but has multiple messages within it, I will try to think of something I like better.

Relatedly, I'd like to bring back `MessageRequest` as the object that creates a `Message`, and remove `toPhone` and `toEmail` from the top level of message since it is redundant and could possibly conflict with the composed objects. This would also clean up the api as shown in swagger:

![image](https://user-images.githubusercontent.com/122047394/235520008-27761eaf-4be4-4f68-9feb-a42a6073ddd6.png)